pulumi/hcl2_type_reflection/hcl2_type_reflection/hcl2_type_reflection.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any
+
+from lark import Lark, Transformer
+
```

Comment:
I'll note that I've only implemented what we're using so far. There are more HCL2 types. If we decide to open-source this as a standalone lib I'll finish implementing them